hack/update-munge-docs.sh

```diff
@@ -21,6 +21,9 @@ set -o pipefail
 KUBE_ROOT=$(dirname "${BASH_SOURCE}")/..
 source "${KUBE_ROOT}/hack/lib/init.sh"
 
+git_upstream=$(kube::util::git_upstream_remote_name)
```

Comment:
I suggest continuing to use the env variable, but defaulting it to this value if it's not set.